src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -430,6 +430,19 @@
                         <Insets bottom="4.0" left="4.0" right="4.0" top="4.0"/>
                       </padding>
                     </HBox>
+                    <HBox spacing="4.0">
+                       <children>
+                          <Label prefWidth="220.0" text="Stop Tabulation Early on Round">
+                             <HBox.margin>
+                                <Insets />
+                             </HBox.margin>
+                          </Label>
+                          <TextField fx:id="textFieldStopTabulationEarly" maxWidth="80.0" />
+                       </children>
```

Comment:
Appreciate the screenshot! Curious why you're adding this extra `children`, `insets`, etc. stuff though? I figured you could just copy what was done for Minimum Vote Threshold above and tweak it slightly.